go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/alron/ginlogr v0.0.4
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/briandowns/spinner v1.18.1
+	github.com/btcsuite/btcd/btcutil v1.1.1
```

Comment:
Started mining, are we? :D

Reply:
Ahah, in every k8s cluster we are going to be deployed!